librz/core/cil.c

```diff
@@ -428,3 +428,53 @@ RZ_IPI void rz_core_analysis_esil_default(RzCore *core) {
 	}
 	rz_core_seek(core, at, true);
 }
+
+RZ_IPI void core_rzil_init(RzCore *core) {
```

Comment:
Is this really needed? I think it is used only in this file and you can probably just directly call `rz_analysis_rzil_setup`.